content/docs/2_cookbook/7_extensions/0_create-pdf-preview-images/cookbook-recipe.txt

```diff
@@ -226,20 +227,21 @@ load([
 ], __DIR__);
 
 Kirby::plugin('cookbook/pdfpreview', [
-    'hooks' => [
-        'file.create:after' => function ($file) {
-            // only create preview for PDF files
-            if ($file->mime() === 'application/pdf') {
-                $options    = [
-                    'quality'    => 100,
-                    'page'       => 0,
-                    'density'    => 300,
-                ];
-                $darkroom = new PdfPreview();
-                $darkroom->process($file->root(), $options);
-            }
-        }
-    ],
+	  'hooks' => [
+	    'file.create:after' => function ($file) {
+	      // only create preview for PDF files
+	      if ($file->mime() === 'application/pdf') {
+	        $options    = [
+	          'quality'    => 100,
+	          'page'       => 0,
+	          'density'    => 300,
+	          'colorspace' => false,
```

Comment:
I only added options that are different from the defaults, adding the colorspace option here is therefore not necessary.

Reply:
are you sure?
Without the colorspace attribute, it crashed in my case.